test/regression.uts

```diff
@@ -277,7 +277,7 @@ finally:
 = Test linehexdump function
 conf_color_theme = conf.color_theme
 conf.color_theme = BlackAndWhite()
-assert(linehexdump(Ether(src="00:01:02:03:04:05"), dump=True) == "FFFFFFFFFFFF0001020304059000 ..............")
+assert linehexdump(Ether(src="00:01:02:03:04:05"), dump=True) == 'ff ff ff ff ff ff 00 01 02 03 04 05 90 00  ..............'
```

Comment:
Looks like a regression. Why are you changing the output?

Reply:
See my comment below. I however just noticed that it should have been capitalized letters. Just fixed that (`%02x` ==> `%02X`)